### DISCUSSION TITLE
Watch out on Libera and OFTC

### DISCUSSION THREAD

**Discussion by cmpunches** [rms-support-letter/rms-support-letter.github.io#5558] [General]
They have some folks in the corporate camp.  I walked into a kline on OFTC and FreeNode last night after being asked about the RMS support letter by Tom Wesley on the OFTC board.

If people know you, might be a good idea to stay low for a few weeks on both networks, and watch out for bully staffers.  In my case I can just fart a proxy when I need to so it doesn't really have much impact but if you have alot of investment in your nick you might want to be aware.

**Comment by 6r1d** [comment]
Thank you, I will be careful.

**Comment by dpocock** [comment]
We need to develop alternatives to these platforms.  This has always been a big part of my work.

This situation is a great opportunity to reflect on the question: what would be the right alternative to IRC?

**Reply by kompowiec** [reply]
> what would be the right alternative to IRC?

Nothing. IRC is like a fax.

**Comment by cmpunches** [comment]
@dpocock Why, another IRC network, of course!

**Comment by nukeop** [comment]
I don't get it, what are you implying is going to happen, and why?

**Reply by cmpunches** [reply]
Lotta these folks go to those networks.  There's major activity on FreeNode in particular in support of this effort.  Debian and OFTC are joined at the hip, and some of the OFTC staff are also FreeNode staff; Freenode is better about this kind of thing than OFTC is but there is now crossover blacklisting type behaviour that doesn't represent community interest, so, this is a friendly reminder not to keep too large a profile in those spaces to avoid problems and conflict.

**Reply by nukeop** [reply]
What kinds of problems, exactly? Are you trying to imply that we should be ashamed of our support, or what?

**Reply by cmpunches** [reply]
I think you're misreading.  No, no one should be ashamed to support; they should avoid bans and easily avoidable politics though.

**Reply by franpoli** [reply]
From my experience, IRC channels are usually friendly places where we tend to speak confidently and carelessly. In these difficult times, it might be wise to be cautious and assert your ideas shamelessly but tactfully. The [GNU kind communication guide](https://www.gnu.org/philosophy/kind-communication.html) might give some good advice on how to behave to avoid escalation.

**Reply by 6r1d** [reply]
It is my personal experience, but I've mostly avoided politics, visited `free-rms` by Leah Rowe and was calm. Everything on Freenode was good so far...

**Reply by cmpunches** [reply]
Yeah, Leah is great.  My kline was lifted on Freenode after this post was circulated by folks, but OFTC is maintaining.  Freenode is interesting because they have a bunch of staff who all fall on different sides of issues as they come up in the community.  OFTC seems to only err on the side of being exclusionist dirtbags.

**Reply by 6r1d** [reply]
> OFTC seems to only err on the side of being exclusionist dirtbags

To be honest, I've heard about OFTC from you for the first time, so I don't know how big or small the community there is.

**Reply by cmpunches** [reply]
Wow did this thread age beautifully.

**Comment by cmpunches** [comment]
Since the drafting of this thread Freenode has changed leadership, and the new leadership is not continuing in these kinds of practices from what I can see.

**Reply by jobbautista9** [reply]
Of course you would be defending the new freenode management. You're a part of it.

No amount of shilling would hide the fact that you guys hijacked hundreds of channels though. And that included the #free-rms channel, which has been neutral from the start of the drama. I had to quickly create an emergency PR to remove all instances of freenode on the READMEs, leaving only libera. The apology afterwards wasn't sincere either, if you were sincere, you would reverse both the channel hijacking (indiscriminately, and all of them) and your stupid policy that channels aren't allowed to inform their users that they've moved to another network or platform. But of course you guys didn't do either of that, and you even closed the policy feedback channel, despite opposition from one of your supporters. Bet you didn't like the feedback you're getting huh? You're correct that cancel culture is cancer and should be opposed, but that doesn't mean you should shut yourself off from feedback!

My experience with the former freenode staff now running libera has been pleasant so far. As a Wikimedia volunteer for years, they've been helpful to the Wikimedia community, which is why it was a no-brainer for the Wikimedia GCs to migrate to libera. I've never felt intimidated by them.

Btw, what happened to those blog posts where you are making numerous accusations against the former freenode staff? Why the 404? Look at RMS. He isn't afraid of speaking out loud his thoughts. If you're a real man, you'd stand firm and defend those accusations, not remove them after publication. Did you remove them because people are calling out your bullshit? Perhaps Jose Rizal's father was right: lawyers are liars.

I won't be surprised if you and @dpocock are the same person. Anyway, enjoy serving that dying hermit kingdom! ;)

**Reply by cmpunches** [reply]
>Of course you would be defending the new freenode management. You're a part of it.

Makes sense.  I'm actually really proud of how this crew has pulled together in such a messed up situation between all the harassment and personal attacks, smears online of the staff, and of Andrew, and they still have found a way to run the place with a more civil mindset than just about any critic of the change that I've encountered so far.  It blows my mind that people under such constant vicious attack to their psyches and reputations can be so resilient and provide warmth in the face of it.  I am deeply inspired by what I've observed here and unsurprised by what I've observed of the people lashing out about it. 

>No amount of shilling would hide the fact that you guys hijacked hundreds of channels though. 

This is like saying someone hijacked their own car.  It's their car.  But I agree the channels thing was rough.  It actually was a technical error in the logic of a bot that was written to enforce our spam policy during a massive bombardment of spam, vicious PR strategies and cyberattacks all endorsing Libera while having to turn our heads the other way to the whisper networks spreading misinformation about how things were really going down to undermine confidence in the network's integrity (and the staff's integrity) to force the change to Libera -- not because most users wanted to -- but because a project admin connected to these people and dependent on these people's support for the exclusionary and abusive behaviours I've been talking about wanted to force those users to retain their ability to continue engaging in those behaviours; the old staff's systematically exclusive practices went on for years and they helped people in various f/oss communities do awful things to some community members for a long, long time.  I was one of the people targeted at one point.  You didn't do that of course.  You're just defending it because you haven't been targeted yet for what will become your strengths.

>And that included the #free-rms channel, which has been neutral from the start of the drama. I had to quickly create an emergency PR to remove all instances of freenode on the READMEs, leaving only libera. The apology afterwards wasn't sincere either, if you were sincere, you would reverse both the channel hijacking (indiscriminately, and all of them) and your stupid policy that channels aren't allowed to inform their users that they've moved to another network or platform. 

1) I'm just a person who volunteers there and I am clearly not working.   As you can see, this is not IRC.  I don't work for you here.  Seek me out on IRC to resolve problems on IRC.
2) That problem can be easily fixed on IRC and I'd be happy to do it.  On IRC.

>My experience with the former freenode staff now running libera has been pleasant so far. 

That's because you're not being bombarded by their toxic campaigns as long as you go with it or don't cause too much noise over it.

> As a Wikimedia volunteer for years, they've been helpful to the Wikimedia community, which is why it was a no-brainer for the Wikimedia GCs to migrate to libera. I've never felt intimidated by them.

Well yeah, how else would they get scandals suppressed by an otherwise neutral publication body for their friends and their circles? You'll eventually find ugly parts of those communities that need to be resolved too, and when you do it will be a turning point for you while you decide whether to risk everything to try to fix it, or leave what you love, or, enable it.  Most people ignore it thinking that's all they're doing, but, it's not -- it facilitates it.

> I've never felt intimidated by them.

Not everyone has the option of doing whatever they say for recognition.

>Btw, what happened to those blog posts where you are making numerous accusations against the former freenode staff? 

It's not time yet for my personal story to come out.  It is also a difficult process emotionally as I'd been moving on from it.  I'm a previous victim of cyberstalking and a set of smear campaigns that are still very personal to me.  Everyone on the staff has been targeted by these people at one point with defamation, threats, and other types of abuse.  Mine was particulaly sustained and damaging.  Most people don't have bases of comparison for those kinds of things.  There's a bigger picture here slowly unfolding.  Imagine how angry a group of abusers would become if the platform they were using to hurt people suddenly empowered their targets and disempowered them at the same time?

>Look at RMS. He isn't afraid of speaking out loud his thoughts.

Honestly, while I highly respect him, I think he was rather weak on that front -- he didn't defend himself and he's still paying the price.  His silence allowed the defamation of him to fester and swell to the point that your [Molly De Blancs, Elana Hashmans, Neil McGoverns](http://techrights.org/2021/04/21/some-people-who-asked-to-be-removed-from-the-slanderous-hate-letter-against-the-fsf-are-still-being-denied-removal-but-not-all/) tried to make opportunity of it.  In his case, he's a famous guy already with alot of smart, established people with influence ready to defend him after many years of being at the core of the f/oss movement.  Not everyone can do that in such a way.  In fact, most people who try to come forward with allegations of lynching or ritual defamation tend to walk right into providing justification for more of the same due to the public's general lack of [education on how those types of abusive campaigns work](https://phanes.silogroup.org/strength-establishment-and-pizza/).     

If I can think of a comparison for your statement, though, I would probably liken it to that of the type of thinking that destroyed the british army during the revolutionary war -- they had rigid movement structures that paced forward in extremely slow and predictable patterns irregardless of the positions and strategies of their enemies, holding to strict rank and file on principle. 

Then they encountered guerrilla warfare and the results were graphic despite lower levels of technological advancement to most adversaries at the time.  You can't treat every adversary with the same rigid response and not expect to lose, just as a strategy for one person to deal with a thing, may not be the best strategy for people in a similar situation but with different resources to draw from.

> If you're a real man, you'd stand firm and defend those accusations, not remove them after publication. 

The appeal to toxic masculinity really gave away the weak footing on this.  Listen, these things are complicated from a liability and reputation management perspective.  People have alot more to worry about and alot less freedom than they think they do when they're younger.  It'll probably come out in time and in the right way.  I'm finding ways.

God I wish I could see the world as I did when I was 18.  Things were so much simpler.  Not because they actually were, but because my understanding of it was still forming and I had no idea because I grew quickly cognitively just like you.  I had not yet seen the loss that people incur from going into battle without a plan against certain types of problems.  As an "older adult", people will do things to you occasionally that you will have to try to let go when you really, really do not want to -- not because it's the right thing to do but because you have to in order to maintain your health and your happiness as a human being.  Sure, people let things go all the time, but sometimes your sense of justice will pull at you, sometimes in a few directions at once. It may be a long time before you understand this and I sincerely hope that is the case, because while there are many great aspects of growing older, just about everything else sucks -- most older personality traits are trauma responses.  Hold onto that feel, man.

>I won't be surprised if you and @dpocock are the same person. Anyway, enjoy serving that dying hermit kingdom! ;)

No he is another target who is being attacked instead of helped over trying to tell his story; if you want to do some good in the world, help that guy get the help he needs to recover and get his story out like I did (or am finding a way to).  These kinds of things can "break" people.

In any case, almost all of this is off-topic, so please accept this consolation photo:

![Clowns Are Funny](https://www.syracuse.com/resizer/FTo1-LlXkalsbVcopkDUXYIwFm8=/1280x0/smart/advancelocal-adapter-image-uploads.s3.amazonaws.com/image.syracuse.com/home/syr-media/width2048/img/us-news/photo/2016/10/06/clowns-b32f08acf0414ded.jpg)

**Reply by nukeop** [reply]
> No he is another target who is being attacked instead of helped over trying to tell his story

His story is that he's a crazy person, a spammer, and possibly your alter ego.
